src/SpineAnimationRenderer.ts

```diff
@@ -333,40 +371,9 @@ export class SpineAnimationRenderer extends Renderer {
     this._dirtyUpdateFlag &= ~type;
   }
 
-  /**
-   * @internal
-   */
-  _onWorldVolumeChanged(): void {
-    this._dirtyUpdateFlag |= RendererUpdateFlags.WorldVolume;
-  }
-
-  private _initialize() {
-    this._applyDefaultConfig();
-    this._dirtyUpdateFlag |= SpineAnimationUpdateFlags.InitialVolume;
-    this._state.addListener({
-      start: () => {
-        this._onAnimationStart();
-      },
-      complete: (entry: TrackEntry) => {
-        this._onAnimationComplete(entry);
-      },
-    });
-    this.update(0);
-  }
-
-  private _onAnimationStart(): void {
-    this._dirtyUpdateFlag |= SpineAnimationUpdateFlags.AnimationVolume;
-  }
-
-  private _onAnimationComplete(entry: TrackEntry): void {
-    if (!entry.loop) {
-      this._setDirtyFlagFalse(SpineAnimationUpdateFlags.AnimationVolume)
-    }
-  }
-
   private _clearMaterialCache(): void {
     this._materials.forEach((item) => {
-      const texture = item.shaderData.getTexture('material_SpineTexture');
+      const texture = item.shaderData.getTexture("material_SpineTexture");
       const blendMode = getBlendMode(item);
```

Comment:
1、Cache SpineAnimationRenderer._materialCache outside the loop
2、rename item to material may be more intuitive

Reply:
done